monkey/monkey_island/cc/consts.py

```diff
@@ -16,3 +16,5 @@
 DEFAULT_LOGGER_CONFIG_PATH = os.path.join(
     MONKEY_ISLAND_ABS_PATH, "cc", "island_logger_default_config.json"
 )
+
+DEFAULT_DATA_DIR = os.path.join(MONKEY_ISLAND_ABS_PATH, "cc")
```

Comment:
Term "Data" is misleading. I'd rename this as `ISLAND_CODE_DIR` or `CC_CODE_DIR`

Reply:
By default it is, only because that's what we're doing today. We need to move away from storing data generated at runtime alongside our code. Since we're so close to a release, I don't want to change that right now, except when running from an AppImage.